packages/harmony/src/components/input/SelectablePill/SelectablePill.tsx

```diff
@@ -16,6 +16,7 @@ export const SelectablePill = forwardRef<
       size = 'default',
       isSelected,
       label,
+      isDisabled,
```

Comment:
I personally prefer `disabled` because it follows the existing convention for html elements, specifically inputs. And it's also already a prop for `button`?

Reply:
yeah fair enough!